libs/agno/agno/app/fastapi/async_router.py

```diff
@@ -270,6 +297,11 @@ async def run_agent_or_team(
                 team.monitoring = True
             else:
                 team.monitoring = False
+        elif workflow:
+            if monitor:
+                workflow.monitoring = True
+            else:
+                workflow.monitoring = False
```

Comment:
`workflow.monitoring = bool(monitor)` would be the same

Reply:
Nice one